.github/workflows/main.yml

```diff
@@ -1,42 +1,22 @@
 on: ["pull_request"]
 name: Static checks
 jobs:
-  test:
-    runs-on: ubuntu-latest
+  static-checks:
+    runs-on: ubuntu-22.04
     env:
-      GO111MODULE: off
-      TRAVIS: "true"
-      TRAVIS_BRANCH: ${{ github.base_ref }}
-      TRAVIS_PULL_REQUEST_BRANCH: ${{ github.head_ref }}
-      TRAVIS_PULL_REQUEST_SHA : ${{ github.event.pull_request.head.sha }}
+      GOPATH: ${{ github.workspace }}
     steps:
-    - name: Install Go
-      uses: actions/setup-go@v2
-      with:
-        go-version: 1.19.3
-    - name: Setup GOPATH
-      run: |
-        gopath_org=$(go env GOPATH)/src/github.com/kata-containers/
-        mkdir -p ${gopath_org}
-        ln -s ${PWD} ${gopath_org}
-        echo "TRAVIS_BRANCH: ${TRAVIS_BRANCH}"
-        echo "TRAVIS_PULL_REQUEST_BRANCH: ${TRAVIS_PULL_REQUEST_BRANCH}"
-        echo "TRAVIS_PULL_REQUEST_SHA: ${TRAVIS_PULL_REQUEST_SHA}"
-        echo "TRAVIS: ${TRAVIS}"
-    - name: Checkout code
-      uses: actions/checkout@v2
-      with:
-        fetch-depth: 0
-    - name: Setup travis references
-      run: |
-        echo "TRAVIS_BRANCH=${TRAVIS_BRANCH:-$(echo $GITHUB_REF | awk 'BEGIN { FS = \"/\" } ; { print $3 }')}" 
-    - name: Setup
-      run: |
-        community_repo=$(go env GOPATH)/src/github.com/kata-containers/community
-        pushd ${community_repo}
-        GOPATH=$(go env GOPATH) .ci/setup.sh 
-    - name: Running static checks
-      run: |
-        community_repo=$(go env GOPATH)/src/github.com/kata-containers/community
-        pushd ${community_repo}
-        GOPATH=$(go env GOPATH) .ci/static-checks.sh
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          path: ./src/github.com/${{ github.repository }}
+
+      - name: Install system dependencies
+        run: |
+          sudo apt-get -y install moreutils hunspell hunspell-en-gb hunspell-en-us pandoc
+
+      - name: Running static checks
+        run: |
+          export PATH=${PATH}:${GOPATH}/bin
```

Comment:
For extra safety, I'd put `GOPATH/bin` first.

Reply:
Done. Thanks!